pkg/syncer/metadatasyncer.go

```diff
@@ -458,26 +458,26 @@ func pvcUpdated(oldObj, newObj interface{}, metadataSyncer *metadataSyncInformer
 		}
 		log.Debugf("PVCUpdated: Found Persistent Volume %s from API server", newPvc.Spec.VolumeName)
 	}
+	migrationEnabled := metadataSyncer.coCommonInterface.IsFSSEnabled(ctx, common.CSIMigration)
 	// Verify if csi migration is ON and check if there is any label update or migrated-to annotation was received for the PVC
-	if pv.Spec.VsphereVolume != nil {
-		if metadataSyncer.coCommonInterface.IsFSSEnabled(ctx, common.CSIMigration) {
-			if oldPvc.Status.Phase == v1.ClaimBound &&
-				reflect.DeepEqual(newPvc.GetAnnotations(), oldPvc.GetAnnotations()) &&
-				reflect.DeepEqual(newPvc.Labels, oldPvc.Labels) {
-				log.Debug("PVCUpdated: PVC labels and annotations have not changed for %s in namespace %s", newPvc.Name, newPvc.Namespace)
+	if migrationEnabled && pv.Spec.VsphereVolume != nil {
+		if !migratedToAnnotationExists(ctx, newPvc.Annotations) {
+			log.Info("migration annotation is not present on the PVC. Skipping update")
```

Comment:
add "PVCUpdated" in the beginning of log message

Reply:
This log message does not say which PVC it is skipping. Could we add that info? And I think this needs to be in debug log message.

Reply:
I think we should print PV/PVC we are dealing with as a debug message in the beginning. 
And then we can use contextual logging feature `TraceId` to identify specific updates.

Reply:
fixed.